doc/source/dev/contributor/quickstart_mac.rst

```diff
@@ -51,7 +51,7 @@ Building SciPy
 
 #. Use Homebrew to install ``gcc`` by entering the command ``brew install gcc``.
 
-#. In the terminal, update/upgrade all of SciPy's build dependencies: ``conda update cython numpy pytest pybind11``
+#. In the terminal, ensure that all of SciPy's build dependencies are up to date: ``conda install pybind11``, then ``conda update cython numpy pytest pybind11``.
```

Comment:
`pybind` is not distributed with Anaconda, so the user will not yet have it.

Reply:
ok